frontend/src/components/license/licenseUtils.spec.ts

```diff
@@ -148,8 +148,8 @@ describe('licenseUtils', () => {
 
       const result = licensesToSimplifiedPreview(licenses);
       expect(result).toEqual([
-        { name: 'Console Community', expiresAt: '' },
-        { name: 'Redpanda Enterprise', expiresAt: '7/15/2122' }, // Based on the expiration timestamp
+        { name: 'Console Community', expiresAt: '', isExpired: false },
```

Comment:
should we have a test for `isExpired: true`?

Reply:
We will be adding unit tests in a different PR, we will cover this as well